parsers/postWithObjectParser.js

```diff
@@ -47,7 +47,7 @@ const createOrUpdatePost = async function ( data ) {
             };
         } );
     }
-    await commentRefSetter.addPostWithWobj( `${data.author }_${ data.permlink}`, data.wobjects );
+    await commentRefSetter.addPostRef( `${data.author }_${ data.permlink}`, data.wobjects );
```

Comment:
`${data.author }_${ data.permlink}` - use (or do not use) spaces after/before braces